src/actinia_core/resources/ephemeral_processing_with_export.py

```diff
@@ -222,7 +222,7 @@ def _export_raster(self, raster_name,
 
         Args:
             raster_name (str): The name of the raster layer
-            format (str): COG
+            format (str): COG (default), GTiff
```

Comment:
COG is only the default if COG is possible in actinia. Otherwise GTiff is the default

Reply:
(see also https://github.com/mundialis/actinia_core/blob/fcaa010576bae41139f674339941bd8d0519ac35/src/actinia_core/resources/ephemeral_processing_with_export.py#L251)

Reply:
> COG is only the default if COG is possible in actinia. Otherwise GTiff is the default

This only depends on GDAL effectively, right? GDAL >= 3.1 offers COG

Reply:
Yes

Reply:
thx, updated accordingly